src/utils/createGenerateClassName.js

```diff
@@ -15,5 +21,23 @@ const jssCounter = globalRef[namespace]++
 export default (): generateClassName => {
   let ruleCounter = 0
 
-  return (rule: Rule): string => `${rule.key}-${jssCounter}-${ruleCounter++}`
+  return (rule: Rule, sheet?: StyleSheet): string => {
+    ruleCounter += 1
+
+    if (ruleCounter > maxRules) {
+      warning(
+        false,
+        'You might have a memory leak. Rule counter is at %s.',
+        ruleCounter
+      )
+    }
+
+    if (env === 'production') {
+      return `c${jssCounter}${ruleCounter}`
```

Comment:
it is configurable, you can pass your own  class name generator function http://cssinjs.org/js-api?v=v9.0.0-pre#generate-your-own-class-names